website/src/documentation/programming-guide.md

```diff
@@ -2907,3 +2907,137 @@ elements, or after a minute.
 ```py
 {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets_test.py tag:model_other_composite_triggers
 %}```
+
+## 9. Metrics {#metrics}
+The Beam model provides a way to use different types of metrics in a user pipeline.  There could be 
```

Comment:
```suggestion
In the Beam model, metrics provide some insight into the current state of a user pipeline, potentially while the pipeline is running.  There could be 
```
(Just a suggestion -- the first sentence should describe why metrics are interesting.  The examples are a good illustration!)